test/e2e/node_feature_discovery_test.go

```diff
@@ -177,7 +177,7 @@ func cleanupCRs(ctx context.Context, cli *nfdclient.Clientset, namespace string)
 }
 
 // Actual test suite
-var _ = SIGDescribe("NFD master and worker", func() {
+var _ = NFDDescribe(Label("e2e-test", "master-and-worker"), func() {
```

Comment:
You added `nfd-worker` below so just use `nfd-master` here
```suggestion
var _ = NFDDescribe(Label("nfd-master"), func() {
```